portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -1087,32 +1057,28 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-47381. User can authorize external applications."
-	@ignore = "true"
 	test ExternalApplicationCanBeAuthorized {
 		property custom.properties = "virtual.hosts.default.site.name=Guest";
 		property oauth2.local.enabled = "true";
 		property portal.acceptance = "true";
-		property portal.ssl.enabled = "true";
 		property test.name.skip.portal.instance = "OAuth2#ExternalApplicationCanBeAuthorized";
 
 		task ("Add and edit an external application") {
-			OAuth2.openSSLPage();
-
 			OAuth2.addAndEditExternalApplication(
 				applicationName = "Third-party Application",
-				baseURL = "https://localhost:8443",
-				callbackUri = '''https://localhost:8443/oauth2-tester/?url=https://localhost:8443&client_id=abc123''',
+				baseURL = "http://localhost:8080",
+				callbackUri = '''http://localhost:3000/liferay-oauth2''',
 				clientId = "abc123",
 				clientProfile = "Web Application",
 				clientSecret = "abc123",
-				websiteURL = "https://localhost:8443");
+				websiteURL = "http://localhost:3000/liferay-oauth2");
 		}
 
 		task ("Authorize the new external application") {
 			OAuth2.authorizeMyApplication(
 				clientId = "abc123",
 				clientSecret = "abc123",
-				portalURL = "https://localhost:8443");
+				portalURL = "http://localhost:3000/liferay-oauth2");
```

Comment:
This line should be removed, otherwise the **Liferay Authorize URL** will be **http://localhost:3000/liferay-oauth2** rather than **http://localhost:8080/o/oauth2/authorize**